src/ExamplePairing/ExamplePairing.tsx

```diff
@@ -31,10 +33,18 @@ class ExamplePairing extends Component<IProps, {}> {
     if (!example || _.isEmpty(example)) {
       return null;
     }
+    const paramStructure: TParamStructure = method?.paramStructure || "either";
```

Comment:
does typescript produce something that works in browsers for this?

`?` isnt all that well supported yet.

Reply:
yea you just `target: es5`